sdks/python/apache_beam/examples/snippets/snippets.py

```diff
@@ -175,8 +177,9 @@ def _add_argparse_args(cls, parser):
        'Or to take arms against a sea of troubles, '])
    | beam.io.WriteToText(my_options.output))
 
-  p.run()
+  result = p.run()
```

Comment:
Seems a lot of these snippets could be updated to use TestPipeline, right? (At least as long as it's part of the setup/teardown code, not the actual snippet.) They're never run not under a test...